src/core/Geometry.js

```diff
@@ -399,6 +399,8 @@ Object.assign( THREE.Geometry.prototype, THREE.EventDispatcher.prototype, {
 
 	computeVertexNormals: function ( areaWeighted ) {
 
+		this.computeFaceNormals();
```

Comment:
Yes. Since face normals are required for the computation, I decided to add that line to both this method and the new `ComputeFlatVertexNormals()`. It was a judgement call.

Hmmm. Now I see normals are only required for the non-area-weighted case...